app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :id, :begin_date, :end_date
+
+  belongs_to :user
+  belongs_to :book
```

Comment:
be careful with this when you are doing an index of the rents, this will trigger N+1 queries. 1 for the fetch of the rents and N for every book in the collection.
Take a look at this https://engineering.gusto.com/a-visual-guide-to-using-includes-in-rails/